src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,62 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { Sorttype } from '../../types/Sorttype';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface Props {
+  sortType: string,
+  setSortType: (value: string) => void,
+  query: string,
+  setQuery: (value: string) => void,
+}
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: React.FC<Props> = ({
+  sortType,
+  setSortType,
+  query,
+  setQuery,
+
+}) => {
+  return (
+
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            value={sortType}
+            onChange={(event) => setSortType(event.target.value)}
+          >
+            <option value="all">{Sorttype.all}</option>
+            <option value="active">{Sorttype.active}</option>
+            <option value="completed">{Sorttype.completed}</option>
```

Comment:
will be better use `.map()` and iterate `option`

Reply:
Could you please specify this moment, as I vaguely get what should be done

Reply:
it's not critical but you can iterate by `enum` SortType